### PR TITLE
Add format to oai_dc response

### LINKED ISSUE #3973: publish dc:format in oai_dc
From gallica's documentation:

**Comply with MIME types**
E.g. &lt;dc:format&gt;image/jpeg&lt;/dc:format&gt;
or &lt;dc:format&gt;application/pdf&lt;/dc:format&gt;

**A MIME type = an element**
If several formats are available or if they give several pieces of
information, repeat the element
- prefer: &lt;dc:format&gt;image/jpeg&lt;/dc:format&gt;
&lt;dc:format&gt;388 Mo &lt;/dc:format&gt; to
&lt;dc:format&gt;image/jpeg ; 388 Mo&lt;/dc:format&gt;

**Source format**
The &lt;dc:format&gt; is used to describe the original physical document
(number of pages, dimensions, etc.)

### PR DESCRIPTION
**Pull request by hackmastera**
fixes #3973

### CODE REVIEW

File: app/models/oai/figgy/oai_wrapper.rb [hunk 1]
```diff
@@ -32,8 +36,22 @@ def sets
...
+      end
+
       def decorated_resource
         __getobj__.decorate
       end
+
+      def mime_types
+        file_sets = decorated_resource.file_sets
+        return file_sets.flat_map(&:mime_type) unless file_sets.empty?
+        Wayfinder.for(resource).deep_file_sets.flat_map(&:mime_type)
```
Comment (tpendragon) [inline app/models/oai/figgy/oai_wrapper.rb:50]: Should these be `uniq`?

File: app/models/oai/figgy/oai_wrapper.rb [hunk 2]
```diff
@@ -32,8 +36,22 @@ def sets
...
+      def resource
+        __getobj__
+      end
+
       def decorated_resource
         __getobj__.decorate
       end
+
+      def mime_types
+        file_sets = decorated_resource.file_sets
```
Comment (tpendragon) [inline app/models/oai/figgy/oai_wrapper.rb:48]: What if the resource doesn't respond to `file_sets`? They might all, I'm unsure.

File: app/models/oai/figgy/oai_wrapper.rb (hunk 1 above)
Comment (hackmastera) [reply app/models/oai/figgy/oai_wrapper.rb:50]: good call!!

File: app/models/oai/figgy/oai_wrapper.rb (hunk 2 above)
Comment (hackmastera) [reply app/models/oai/figgy/oai_wrapper.rb:48]: Seems like a perfectly reasonable case to hedge against.

Comment (hackmastera) [reply app/models/oai/figgy/oai_wrapper.rb:52]: Added a test for this.

**Comment by hackmastera** [comment]
@tpendragon ready for another look

**Pull request merged**